plugins/toolkit/src/api.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createApiRef,
+  DiscoveryApi,
+  FetchApi,
+  IdentityApi,
+} from '@backstage/core-plugin-api';
+
+export const toolkitApiRef = createApiRef<ToolkitApi>({
+  id: 'toolkit',
+});
+
+export interface ToolkitApi {
+  createToolkit(body: any): Promise<any>;
```

Comment:
as in my previous comment it would be better to be more specific with typings here

Reply:
Fixed